aip/client-libraries/4290.md

```diff
@@ -36,8 +36,8 @@ the proto import root, on a POSIX machine):
 
 ```bash
 $ docker run --rm --user $UID \
-  --mount type=bind,source=a/b/c/v1/,destination=/in/a/b/c/v1/,readonly \
-  --mount type=bind,source=dest/,destination=/out/ \
+  --mount type=bind,source=`pwd`/a/b/c/v1/,destination=/in/a/b/c/v1/,readonly \
+  --mount type=bind,source=/path/to/dest/,destination=/out/ \
```

Comment:
I think this second line should remain the same.

Reply:
My main concern is that it won't work when copied and pasted.

```sh
$ mkdir -p dest && \
  docker run \
    --mount type=bind,source=/Users/fenster/googleapis/google/devtools/cloudbuild/v1,destination=/in/google/devtools/cloudbuild/v1,readonly \
    --mount type=bind,source=dest/,destination=/out \
    --rm gapic-generator-typescript
docker: Error response from daemon: invalid mount config for type "bind": invalid mount path: 'dest/' mount path must be absolute.
See 'docker run --help'.
```

Reply:
Oh, I see. I should have read better. It is the source directory. Sure.